lib/models/docker-image-builder.ts

```diff
@@ -77,6 +79,7 @@ export class DockerImageBuilder extends Construct {
         ec2InstanceProfileRole.attachInlinePolicy(ec2InstanceProfilePolicy);
 
         const role = new Role(this, createCdkId([stackName, 'docker_image_builder_role']), {
+            roleName: createCdkId([stackName, 'docker_image_builder_role']),
```

Comment:
not anything we need to address now, but as a backlog item or something later, we should standardize name formats. we have every example possible between capitals, dashes, underscores, and so on for logical and physical names